src/components/Thumbnail/Name.tsx

```diff
@@ -1,5 +1,12 @@
 import React, { FC } from "react";
+import classnames from "classnames";
 
-export const Name: FC = ({ children }) => (
-  <p className="font-gilbert text-21 text-black w-full">{children}</p>
+interface IProps {
+  className?: string;
+}
+
+export const Name: FC<IProps> = ({ children, className }) => (
+  <h5 className={classnames("text-21 text-black font-bold", className)}>
```

Comment:
style not actually like at Zeplin, fix that

Reply:
discussed